packages/cli/openapi-ir-to-fern/src/converters/convertToServices.ts

```diff
@@ -47,10 +50,11 @@ export function convertToServices({
         const service = services[file];
         if (service != null) {
             if (endpointId in service.value.endpoints) {
-                throw new Error(
-                    `The OpenAPI Spec has conflicting sdk method names. See ${endpoint.method.toUpperCase()} ${
+                const firstEnd = endpoints.find((e) => e.sdkName?.methodName === endpointId);
+                context.logger.error(
+                    `The OpenAPI Spec has conflicting SDK method names. See ${endpoint.method.toUpperCase()} ${
                         endpoint.path
-                    }`
+                    }, ${firstEnd?.method.toUpperCase()} ${firstEnd?.path} already has this method name.`
```

Comment:
Should we log all the conflicting methods + paths, it might be confusing to only log one?